satellite/satellitedb/migrate.go

```diff
@@ -1150,221 +375,474 @@ func (db *DB) PostgresMigration() *migrate.Migration {
-					`ALTER TABLE graceful_exit_transfer_queue ALTER COLUMN requested_at TYPE timestamp;`,
-					`ALTER TABLE graceful_exit_transfer_queue ALTER COLUMN last_failed_at TYPE timestamp;`,
-					`ALTER TABLE graceful_exit_transfer_queue ALTER COLUMN finished_at TYPE timestamp;`,
-				},
+				Action:      migrate.SQL{},
 			},
 			{
 				DB:          db.db,
 				Description: "Add table for storing stripe customers",
 				Version:     59,
-				Action: migrate.SQL{
-					`DROP TABLE project_payments CASCADE`,
-					`DROP TABLE user_payments CASCADE`,
-					`CREATE TABLE stripe_customers (
-						user_id bytea NOT NULL,
-						customer_id text NOT NULL,
-						created_at timestamp with time zone NOT NULL,
-						PRIMARY KEY ( user_id ),
-						UNIQUE ( customer_id )
-					);`,
-				},
+				Action:      migrate.SQL{},
 			},
 			{
 				DB:          db.db,
 				Description: "Add coinpayments_transactions table",
 				Version:     60,
-				Action: migrate.SQL{
-					`CREATE TABLE coinpayments_transactions (
-						id text NOT NULL,
-						user_id bytea NOT NULL,
-						address text NOT NULL,
-						amount bytea NOT NULL,
-						received bytea NOT NULL,
-						status integer NOT NULL,
-						key text NOT NULL,
-						created_at timestamp with time zone NOT NULL,
-						PRIMARY KEY ( id )
-					);`,
-				},
+				Action:      migrate.SQL{},
 			},
 			{
 				DB:          db.db,
 				Description: "Add graceful exit success column to nodes table",
 				Version:     61,
-				Action: migrate.SQL{
-					`ALTER TABLE nodes ADD COLUMN exit_success boolean NOT NULL DEFAULT FALSE`,
-				},
+				Action:      migrate.SQL{},
 			},
 			{
 				DB:          db.db,
 				Description: "Alter graceful_exit_transfer_queue to have the piece_num as part of the primary key since it is possible for a node to have 2 pieces for a given segment.",
 				Version:     62,
-				Action: migrate.SQL{
-					`ALTER TABLE graceful_exit_transfer_queue DROP CONSTRAINT graceful_exit_transfer_queue_pkey;`,
-					`ALTER TABLE graceful_exit_transfer_queue ADD PRIMARY KEY ( node_id, path, piece_num );`,
-				},
+				Action:      migrate.SQL{},
 			},
 			{
 				DB:          db.db,
 				Description: "Add payments update balance intents",
 				Version:     63,
-				Action: migrate.SQL{
-					`CREATE TABLE stripecoinpayments_apply_balance_intents (
-						tx_id text NOT NULL REFERENCES coinpayments_transactions( id ) ON DELETE CASCADE,
-						state integer NOT NULL,
-						created_at timestamp with time zone NOT NULL,
-						PRIMARY KEY ( tx_id )
-					);`,
-				},
+				Action:      migrate.SQL{},
 			},
 			{
 				DB:          db.db,
 				Description: "Removing unused bucket_usages table",
 				Version:     64,
-				Action: migrate.SQL{
-					`DROP TABLE bucket_usages CASCADE;`,
-				},
+				Action:      migrate.SQL{},
 			},
 			{
 				DB:          db.db,
 				Description: "Add stripecoinpayments_invoice_project_records",
 				Version:     65,
+				Action:      migrate.SQL{},
```

Comment:
has this been already depoloyed to production?

Reply:
not sure, i imagine it will be by the time this is merged. If not, i can remove

Reply:
oh but in the meantime this could break the backwards compatibility tests. let me investigate

Reply:
confirmed the last release tag `v0.25.1` goes up to migration v65.  I updated here to reflect that.